include/live_model.h

```diff
@@ -79,9 +80,10 @@ namespace reinforcement_learning {
      * Initialize the library and start the background threads used for
      * model managment and sending actions and outcomes to the online trainer
      * @param status  Optional field with detailed string description if there is an error
+     * @param trace_callback Optional trace logger callback
      * @return int Return error code.  This will also be returned in the api_status object
      */
-    int init(api_status* status=nullptr);
+    int init(api_status* status=nullptr, trace_callback binding_trace=nullptr);
```

Comment:
>binding_trace [](start = 56, length = 13)

binding_trace is exposing the internal functionality. Please use a naming that is closer to "trace_callback".

Reply:
renamed to "binding_trace_callback"

---
In reply to: [252729441](https://github.com/VowpalWabbit/reinforcement_learning/pull/38#discussion_r252729441) [](ancestors = 252729441)